File: bidding-engine/src/main/java/com/omnibus/bidding/events/RecommendationProducedEvent.java
package com.omnibus.bidding.events;

import java.time.Instant;
import java.util.List;

public record RecommendationProducedEvent(
    String eventId,
    Instant occurredAt,
    String system,
    String evaluatedSeat,
    String northHand,
    String southHand,
    String auction,
    String recommendedBid,
    String explanation,
    List<String> candidateBids
) {
}

File: services/bidding-engine/src/main/kotlin/com/omnibus/bidding/events/RecommendationProducedEvent.kt
package com.omnibus.bidding.events

import java.time.Instant

data class RecommendationProducedEvent(
    val eventId: String,
    val occurredAt: Instant,
    val system: String,
    val evaluatedSeat: String,
    val northHand: String,
    val southHand: String,
    val auction: String,
    val recommendedBid: String,
    val explanation: String,
    val candidateBids: List<String>,
)

